module/applications/actor/character-generation-prompt.mjs

```diff
@@ -97,39 +120,46 @@ export default class CharacterGenerationPrompt extends HandlebarsApplicationMixi
       classes:  [ "navigation" ],
     },
     "namegiver-tab": {
-      template: "systems/ed4e/templates/actor/generation/namegiver-selection.hbs",
-      id:       "-namegiver-tab",
-      classes:  [ "namegiver" ],
+      template:   "systems/ed4e/templates/actor/generation/namegiver-selection.hbs",
+      id:         "-namegiver-tab",
+      classes:    [ "namegiver" ],
+      scrollable: [ "" ],
     },
     "class-tab": {
-      template: "systems/ed4e/templates/actor/generation/class-selection.hbs",
-      id:       "-class-tab",
-      classes:  [ "class" ],
+      template:   "systems/ed4e/templates/actor/generation/class-selection.hbs",
+      id:         "-class-tab",
+      classes:    [ "class" ],
+      scrollable: [ "" ],
     },
     "attribute-tab": {
-      template: "systems/ed4e/templates/actor/generation/attribute-assignment.hbs",
-      id:       "-attribute-tab",
-      classes:  [ "attribute" ],
+      template:   "systems/ed4e/templates/actor/generation/attribute-assignment.hbs",
+      id:         "-attribute-tab",
+      classes:    [ "attribute" ],
+      scrollable: [ "" ],
     },
     "spell-tab": {
-      template: "systems/ed4e/templates/actor/generation/spell-selection.hbs",
-      id:       "-spell-tab",
-      classes:  [ "spell" ],
+      template:   "systems/ed4e/templates/actor/generation/spell-selection.hbs",
+      id:         "-spell-tab",
+      classes:    [ "spell" ],
+      scrollable: [ "" ],
     },
     "skill-tab": {
-      template: "systems/ed4e/templates/actor/generation/skill-selection.hbs",
-      id:       "-skill-tab",
-      classes:  [ "skill" ],
+      template:   "systems/ed4e/templates/actor/generation/skill-selection.hbs",
+      id:         "-skill-tab",
+      classes:    [ "skill" ],
+      scrollable: [ "" ],
     },
     "language-tab": {
-      template: "systems/ed4e/templates/actor/generation/language-selection.hbs",
-      id:       "-language-tab",
-      classes:  [ "language" ],
+      template:   "systems/ed4e/templates/actor/generation/language-selection.hbs",
+      id:         "-language-tab",
+      classes:    [ "language" ],
+      scrollable: [ "" ],
     },
     "equipment-tab": {
-      template: "systems/ed4e/templates/actor/generation/equipment.hbs",
-      id:       "-equipment-tab",
-      classes:  [ "equipment" ],
+      template:   "systems/ed4e/templates/actor/generation/equipment.hbs",
+      id:         "-equipment-tab",
+      classes:    [ "equipment" ],
+      scrollable: [ "" ],
```

Comment:
Why make the tabs scrollable instead of the window big enough to contain everything? If scrolling can be avoided, I think thats a good thing. Forces you to focus on the most important parts? (Just talking about the tabs themselves, not long lists of stuff like talents or so on...)

Reply:
I disagree. the frame has a width of 1000 which is slightly below tablet width and the height is auto, but nevertheless, if the screen is smaller, it will be editable. Good thing about this scrollable feature is, that it only appears, if the dialog is on overflow. Usually there is no scrollbar